proto/enums/enums.proto

```diff
@@ -412,6 +412,10 @@ enum StatusCode {
     INSPECT_UNKNOWN_ERR = 1301;
     // Trying to add new build environment that already exists
     INSPECT_BUILD_ENV_ALREADY_EXISTS_ERR = 1302;
+    // Trying to modify build environment that doesn't exist
+    INSPECT_BUILD_ENV_NOT_FOUND_ERR = 1303;
```

Comment:
A profile can only have 1 build env set right? By default its local?
So, this error code should be `INSPECT_BUILD_ENV_INCORRECT_TYPE` or trying to modify a build env of incorrect type?